eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -151,14 +157,54 @@ stages:
                   deploy:
                     steps:
                       - checkout: self
+                      - template: /eng/pipelines/templates/steps/stage-artifacts.yml
+                        parameters:
+                          SourceFolder: ${{parameters.ArtifactName}}-signed
```

Comment:
We should consider making "-signed" as a default in the template so we don't accidently forget to pass it.

Reply:
Fair point, but it's super minor and I'd rather not retest this again.